### PR TITLE
chore: Configure `prettier-solidity` and run `prettier-solidity` [TKR-532]

### PR DESCRIPTION
**Pull request by eobbad**
## Checklist:

-   [ ] Prefix PR title with `[WIP]` if necessary.
-   [ ] Add tests to cover changes as needed.
-   [ ] Update documentation as needed.
-   [ ] Add new entries to the relevant CHANGELOG.jsons.

### CODE REVIEW

File: .prettierrc [hunk 1]
```diff
@@ -4,5 +4,17 @@
     "singleQuote": true,
     "trailingComma": "all",
     "bracketSpacing": true,
-    "arrowParens": "avoid"
+    "arrowParens": "avoid",
+    "overrides": [
+        {
+          "files": "**/*.sol",
+          "options": {
+            "printWidth": 80,
```
Comment (kyu-c) [inline .prettierrc:12]: 120?

**Review by kyu-c — CHANGES_REQUESTED** [review]

File: .prettierrc (hunk 1 above)
Comment (eobbad) [reply .prettierrc:12]: do you want it 120? This was the default config on the prettier-solidity website.

Comment (kyu-c) [reply .prettierrc:12]: yeah 120!!! (not what i want but that's what we use haha)

Comment (eobbad) [reply .prettierrc:12]: sg will change 👍

**Review by kyu-c — APPROVED** [review]
LGTM -- let's wait for a confirmation from @dekz

**Pull request merged**